File: 2022 czerwiec/python/2.py
with open("liczby.txt", "r") as f:
    file = f.read().strip().split('\n')

best = [0, 0]
for s in file:
    if abs(int(s) - int(s[::-1])) > best[1]:
        best = [s, abs(int(s) - int(s[::-1]))]
print(*best)

File: 2022 czerwiec/c++/2.cpp
#include <iostream>
#include <fstream>
#include <string>
#include <algorithm>
using namespace std;


int main()
{
    ifstream in("liczby.txt");
    
    string s;
    pair <int, int> ans = {0, 0};
    while (in >> s)
    {
        int n = stoi(s);
        reverse(s.begin(), s.end());
        if (abs(n - stoi(s)) > ans.second)
            ans = {n, abs(n - stoi(s))};
    }
    cout << ans.first << ' ' << ans.second << '\n';
}